docker/files/run/run.sh

```diff
@@ -41,40 +72,30 @@ function apply_configuration() {
   else
     diagcfg telemetry disable
   fi
-}
 
-function catchup() {
-  local FAST_CATCHUP_URL="https://algorand-catchpoints.s3.us-east-2.amazonaws.com/channel/CHANNEL/latest.catchpoint"
-  local CATCHPOINT=$(curl -s ${FAST_CATCHUP_URL/CHANNEL/$NETWORK})
-  if [[ "$(echo $CATCHPOINT | wc -l | tr -d ' ')" != "1" ]]; then
-    echo "Problem starting fast catchup."
-    exit 1
+  # start kmd
+  if [ "$START_KMD" = "1" ]; then
+    local KMD_DIR="kmd-v0.5"
+    # on intial bootstrap, this directory won't exist.
+    mkdir -p "$KMD_DIR"
+    if [ -f "/etc/kmd_config.json" ]; then
+      cp /etc/kmd_config.json kmd_config.json
+    else
+      echo "{ \"address\":\"0.0.0.0:${KMD_PORT}\", \"allowed_origins\":[\"*\"] }" >kmd_config.json
+    fi
+
+    if [ "$KMD_TOKEN" = "" ] && [ "$ADMIN_TOKEN" != "" ]; then
+      echo "$ADMIN_TOKEN" >kmd.token
+    else
+      echo "$KMD_TOKEN" >kmd.token
+    fi
```

Comment:
Generally speaking, I'm not a big fan of these sorts of fallback configurations.

Suggestions:
* Remove KMD_TOKEN, document that ADMIN_TOKEN is for the admin API endpoints and KMD endpoints.
* Skip the override and require users to set all 3 tokens separately.

I don't have a strong preference either way. For the former, the `ADMIN_TOKEN` needs a doc update saying is also used for KMD.